client/components/migration/feature-announcement.jsx

```diff
@@ -18,17 +18,26 @@ import {
 	wcshippingMigrationState,
 } from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
 import { installAndActivatePlugins } from './migration-runner';
+import { TIME_TO_REMMEMBER_DISMISSAL_SECONDS } from './constants';
 
 const FeatureAnnouncement = ( { translate, isEligable, previousMigrationState } ) => {
 	const [isOpen, setIsOpen] = useState(isEligable);
 	const [isUpdating, setIsUpdating] = useState(false);
 
+	useEffect(() => {
```

Comment:
If we click "Maybe later" to close the modal, close the label printing modal, then re-open label printing modal, the feature announcement will show up again. This is because the `props` `isEligable` hasn't refresh from PHP yet.

This side effect checks against the cookie when it mounts so that it doesn't reopen the feature announcement modal.